<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>当前数值：</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>白字(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_value_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_value_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>黄字(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_value_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>爆伤(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_value_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最终(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_value_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>三攻(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_value_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>力智(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>当前装备</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QComboBox" name="com_type_1">
      <item>
       <property name="text">
        <string>白字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黄字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>爆伤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最终</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三攻</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>力智</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>裤子</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="com_type_4">
      <item>
       <property name="text">
        <string>白字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黄字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>爆伤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最终</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三攻</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>力智</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="equ_value_2"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="com_type_3">
      <item>
       <property name="text">
        <string>白字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黄字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>爆伤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最终</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三攻</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>力智</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="com_type_2">
      <item>
       <property name="text">
        <string>白字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黄字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>爆伤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最终</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三攻</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>力智</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>戒指</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="equ_max_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>左槽</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="equ_value_1">
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>武器</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>类型</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>当前数值(%)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>最大值(%)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="equ_value_3"/>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="equ_max_3"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="equ_value_4"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="equ_max_4"/>
    </item>
    <item row="1" column="3">
     <widget class="QComboBox" name="equ_max_1">
      <item>
       <property name="text">
        <string>16+Lv2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>计算结果</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="1">
     <widget class="QComboBox" name="res_type_3">
      <item>
       <property name="text">
        <string>白字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黄字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>爆伤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最终</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三攻</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>力智</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="res_type_1">
      <item>
       <property name="text">
        <string>白字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黄字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>爆伤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最终</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三攻</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>力智</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="res_type_2">
      <item>
       <property name="text">
        <string>白字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黄字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>爆伤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最终</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三攻</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>力智</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>武器</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>左槽</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="res_type_4">
      <item>
       <property name="text">
        <string>白字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黄字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>爆伤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最终</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三攻</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>力智</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>戒指</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="res_value_3">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="res_value_2">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="res_value_4">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>裤子</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="res_value_1">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>光环</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="res_type_5">
      <item>
       <property name="text">
        <string>白字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黄字</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>爆伤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最终</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三攻</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>力智</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_halo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>光环</string>
   </property>
  </widget>
  <widget class="QComboBox" name="com_type_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>无</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黄字 + 5%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>爆伤 + 5%</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三攻 + 5%</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="check_halo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>结果数值：</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>白字(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="out_value_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="out_value_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>黄字(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="out_value_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>爆伤(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="out_value_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最终(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="out_value_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>三攻(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="out_value_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>力智(%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
